src/mgr/ActivePyModules.cc

```diff
@@ -1355,9 +1355,10 @@ PyObject *ActivePyModules::get_osd_perf_counters(MetricQueryID query_id)
 
 MetricQueryID ActivePyModules::add_mds_perf_query(
     const MDSPerfMetricQuery &query,
-    const std::optional<MDSPerfMetricLimit> &limit)
+    const std::optional<MDSPerfMetricLimit> &limit,
+    const bool reregister)
```

Comment:
nit: force_register?

Reply:
I think `reregister` should be appropriate word for the new parameters. We are not 'forcing' anything here, but just addressing something (handle the reregister event in `add_query()`) that was missing.

Reply:
`reregister` makes sense from mgr/stats context, however, in ceph-mgr context, it's "forcing" the register to be done unconditionally.

Reply:
Fixed the occurrences which are not specific to the `add_mds_perf_query` context. I hope that's better.